package/doc/sphinx/source/documentation_pages/coordinates/pickle_readers.rst

```diff
@@ -0,0 +1,84 @@
+.. Contains the formatted docstrings for the serialization of universe located 
+.. mainly in 'MDAnalysis/libs/pickle_file_io.py'
+.. _serialization:
+
+*********************************************************
+Serialization of Coordinate Readers
+*********************************************************
+
+To achieve a flawless implementation of parallelism, this document illustrates
+the basic idea of how different coordinate readers are being serialized in MDAnalysis,
+and what developers should do to serialize a new reader.
+
+To make sure every Trajectory reader can be successfully
+serialized, we implement picklable I/O classes (see :ref:`implemented-fileio`).
+When the file is pickled, filename and other necessary attributes of the open 
+file handle are saved. On unpickling, the file is opened by filename.
+This means that for a successful unpickle, the original file still has to
+be accessible with its filename. To retain the current frame of the trajectory,
+:func:`_read_frame(previous frame)` will be called during unpickling.
+
+.. _how_to_serialize_a_new_reader:
+
+How to serialize a new reader
+-----------------------------
+
+File Access
+^^^^^^^^^^^
+If the new reader uses :func:`util.anyopen()` (e.g. PDB), the reading handler
```

Comment:
It would be useful here, for PDB, GSD and GSDPicklable, etc... to add links to the relevant sections. I.e. :class:`MDAnalysis.coordinates.PDB.PDBReader`. I realise it makes things a bit messier in-text though.